bucket/nettraffic.json

```diff
@@ -0,0 +1,29 @@
+{
+    "homepage": "https://www.venea.net/web/nettraffic",
+    "description": "Essential network bandwidth usage monitor",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.venea.net/web/nettraffic#license"
+    },
+    "version": "1.64.0",
+    "url": "https://files02.tchspt.com/storage2/temp/NetTraffic1.64.0.zip",
```

Comment:
Is there any proof that this is permalink will last for longer period of time? I pretty doubt based on the `temp` and assumption of load balancer existence on techpost site.

Reply:
Well, up to now, there's no problem and there's no other better method. If someone provides a more reliable link, we can change it at any time.